tox.ini

```diff
@@ -131,7 +177,68 @@ basepython = python2.7
 
 commands =
     "{toxinidir}/.travis/environment"
-    {toxinidir}/.travis/twistedchecker-diff {posargs:klein}
+
+    "{toxinidir}/.travis/twistedchecker-diff" {posargs:klein}
+
+
+
+##
+# Combine coverage reports
+##
+
+[testenv:coverage_combine]
+
+basepython = python3.5
+
+skip_install = True
+
+deps = coverage
+
+
+whitelist_externals =
+    cp
+
+
+commands =
+    "{toxinidir}/.travis/environment"
+
+    coverage combine --append
+
+    # Copy aside coverage data for each test environment in case we want to look at it later
+    cp "{env:COVERAGE_FILE}" "{envlogdir}/coverage"
```

Comment:
Doesn't this just copy the coverage file from the `coverage_combine` environment?  i.e. nothing?